internal/sys/fs.go

```diff
@@ -34,7 +34,7 @@ const (
 
 const (
 	modeDevice     = uint32(fs.ModeDevice | 0o640)
-	modeCharDevice = uint32(fs.ModeCharDevice | 0o640)
+	modeCharDevice = uint32(fs.ModeDevice | fs.ModeCharDevice | 0o640)
```

Comment:
This subtle construct of requiring both ModeDevice and ModeCharDevice has bitten me so many times.